source/codegen/metadata/nidaqmx/config.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+config = {
+    'api_version': '20.0.0',
+    'c_header': 'NIDAQmx.h',
+    'c_function_prefix': 'DAQmx',
+    'service_class_prefix': 'NiDAQmx',
```

Comment:
@reckenro - You're more familiar with the naming we used than I am. Should this be NiDaqmx instead of NiDAQmx?

Reply:
I believe we've been using the same capitalization the `c_header` uses except "NI" => "Ni". So this seems fine. For example, that's what we've done with the tclk driver: its `service_class_prefix` capitilaztion follows its c_header: https://github.com/ni/grpc-device/blob/main/source/codegen/metadata/nitclk/config.py#L7

Reply:
Interesting. Good question. The "little mx" always trips me up when I try to recapitalize DAQ and I fall back to the brand-name capitalization (or all-lower). `NiDCPower` also matches my choice on the allcaps "DC". So, I'll go with Ryan's rationale and leave it as-is.